linera-views-derive/src/snapshots/linera_views_derive__tests__generate_crypto_hash_code-2.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: linera-views-derive/src/lib.rs
+assertion_line: 793
+expression: generate_crypto_hash_code(input)
+---
+# [async_trait :: async_trait] impl linera_views :: views :: CryptoHashView < CustomContext > for TestView { async fn crypto_hash (& self) -> Result < linera_base :: crypto :: CryptoHash , linera_views :: views :: ViewError > { use linera_base :: crypto :: { BcsHashable , CryptoHash } ; use linera_views :: { batch :: Batch , generic_array :: GenericArray , sha3 :: { digest :: OutputSizeUser , Sha3_256 } , views :: HashableView , } ; use serde :: { Serialize , Deserialize } ; # [derive (Serialize , Deserialize)] struct TestViewHash (GenericArray < u8 , < Sha3_256 as OutputSizeUser > :: OutputSize >) ; impl BcsHashable for TestViewHash { } let hash = self . hash () . await ? ; Ok (CryptoHash :: new (& TestViewHash (hash))) } }
```

Comment:
Any idea how to make this readable though?

Reply:
I've run it through [`prettyplease`](https://github.com/dtolnay/prettyplease).  That should make it a lot more readable :)